tools/generator/src/Generator+AddTargets.swift

```diff
@@ -192,8 +192,8 @@ File "\(headerFile.filePath)" not found in `files`
         let forcedBazelCompileFiles = outputs
             .forcedBazelCompileFiles(buildMode: buildMode)
         let sources = forcedBazelCompileFiles.map(SourceFile.init) +
-            inputs.srcs.map(SourceFile.init) +
-            inputs.nonArcSrcs.map { filePath in
+            inputs.srcs.filter { !$0.path.isHeader }.map(SourceFile.init) +
```

Comment:
Minor optional nit, this could be nicer as an extension:

```swift
private extension Sequence where Element == FilePath {
    var excludingHeaders: [Element] {
        self.filter { !$0.path.isHeader }
    }
}
```